packages/moonstone/FormCheckboxItem/FormCheckboxItem.module.less

```diff
@@ -24,6 +24,15 @@
 		overflow: hidden;
 	}
 
+	.itemIcon {
+		display: inline-block;
+
+		* {
```

Comment:
Good catch! Can we be more conservative and only affect children with `> *` as is done in SlotItem?

Reply:
I was concerned about nested elements, say you have a DIV or a SPAN with icons in it. Those icons wouldn't get trimmed and you wouldn't know why they don't fit. DIV and SPAN may still be commonly used if developers are unaware of `React.createFragment`.

Reply:
Similarly, if devs put non-`moonstone/Icon`s there, every node would lose its margins which would be equally strange.